deps/rabbit/scripts/rabbitmq-server

```diff
@@ -83,7 +83,7 @@ start_rabbitmq_server() {
 stop_rabbitmq_server() {
     if test "$rabbitmq_server_pid"; then
         kill -TERM "$rabbitmq_server_pid"
-        wait "$rabbitmq_server_pid" || true
+        wait "$rabbitmq_server_pid" || (exit $?)
```

Comment:
```suggestion
        wait "$rabbitmq_server_pid" || exit $?
```

Reply:
It'll work in this particular case but we should keep in mind a couple of things:
- with a subshell we only propagate exit code but don't change the flow by enforcing exiting
- in a general case functions do returns and not exit directly so that the flow could be controlled outside of it, e.g. it would still be possible to mask or catch exit code if necessary like this:
```shell
stop_rabbitmq_server || true
# or
exit_code=0
stop_rabbitmq_server || exit_code=$?
echo "$exit_code"
```
Both points don't matter much here though as long as we are aware of this behavior when changing the script in the future